pkg/generator/truststore.go

```diff
@@ -90,6 +90,11 @@ func (ts *TrustStore) EnsureSecretManager(context context.Context, sm secretsman
 
 // Generate  and all of its aliases
 func (ts *TrustStore) Generate() error {
+	// PEM certs - only support adding trusstore paths
+	if ts.V1Spec.PEMFormat {
+		ts.Value = append(ts.Value, ts.refData...)
```

Comment:
Need to add a webhook check to ensure `len(ts.refData)>0` if `ts.V1Spec.PEMFormat==true`

Reply:
We already make sure there's a path set.